README.md

```diff
@@ -30,7 +32,7 @@ This repo can be cloned using [Git](https://git-scm.com/) to obtain the latest d
 
 3. Open a terminal on your system.
```

Comment:
I suggest to use either
3. Open a terminal 
or
3. Open a terminal on user system.

Use "User" instead or "you/your".